src/tests/test_main.py

```diff
@@ -256,7 +269,7 @@ def tod(d):
                 "uri": main.cdnlink("00666", "elife-00666-video4-data1.xlsx"),
                 "filename": "elife-00666-video4-data1.xlsx"
             }],
-            "image": "https://static-movie-usa.glencoesoftware.com/jpg/10.7554/657/f42a609b0e61fc41798dcba3cc0c87598bd2cf9f/elife-00666-video1.jpg",
+            "image": main.iiiflink("00666", "elife-00666-video1.jpg"),
```

Comment:
is journal capable of seeing that this is a IIIF link and as such it should be added `/full/full/0/default.jpg` if directly put into an `<img>` tag (otherwise it returns 30X to `info.json`)? Otherwise the new article JSON may break journal figure pages?
/cc @thewilkybarkid